conda-envs/environment-test-py39.yml

```diff
@@ -0,0 +1,10 @@
+name: pymc-test-py39
```

Comment:
maybe let's just test on py39 and not the others.

Reply:
I dropped 3.7 (we should also do it in PyMC, following numpy schedule). We should try to keep the test here and in pymc (approximately) in sync. As that will help to reduce the friction of moving code from here to pymc. And possible reduce issues with users using both pymc and pymc-experimental.

Reply:
Agreed on all points.